README.md

```diff
@@ -190,22 +191,28 @@ The [Astro Deno](https://github.com/withastro/astro/tree/main/examples/deno)
 example includes a `preview` command that runs the entry script directly. Run
 `npm run build` then `npm run preview` to run the production deno server.
 
-## Troubleshooting
-
-For help, check out the `#support` channel on
-[Discord](https://astro.build/chat). Our friendly Support Squad members are here
-to help!
+## Contributing
 
-You can also check our [Astro Integration Documentation][astro-integration] for
-more on integrations.
+To configure your development environment, clone the repository and install
+[`pnpm`](https://pnpm.io/). `pnpm` is a package manager that emphasizes disk
```

Comment:
Do you prefer to use pnpm? I'm personally in favor of getting rid of pnpm settings (Currently the repo has both `pnpm-lock.yaml` and `package-lock.json` and that feels confusing to me)

Reply:
There is also a `deno.lock` file ;). I guess we don't need the `package-lock.json`? Should we remove that one, assuming that we really do need `pnpm` to build this project right now?

Reply:
Ok. Let's revisit that in later PRs